impl/client/src/main/javascript/web/pentaho/visual/config.js

```diff
@@ -834,7 +834,9 @@ define(function() {
         },
         apply: {
           extension: {
-            valuesVisible: true
+            valuesVisible: true,
+            legendAreaVisible: true,
+            axisGrid: true,
```

Comment:
Please, remove the dangling comma.
Better still, I don't think the tree map supports this property `axisGrid`, so bettter remove it, if that's the case.